documents/storage-format.md

```diff
@@ -37,21 +37,21 @@ As of optimization techniques, for continious input channels like _mouse positio
 | left click | right click | middle click |
 
 ```SQL
-time: TEXT ("YYYY-MM-DD HH:MM:SS.SSS");
-key_code: INTEGER;
+time: INTEGER;
+button: TEXT;
 ```
 
 ### Cursor position change
 
 ```SQL
-time: TEXT ("YYYY-MM-DD HH:MM:SS.SSS");
-cursor_position_x: INTEGER;
-cursor_position_y: INTEGER;
+time: INTEGER;
+x: INTEGER;
+y: INTEGER;
 ```
 
 ### Wheel scroll
 
 ```SQL
-time: TEXT ("YYYY-MM-DD HH:MM:SS.SSS");
-scroll_delta: REAL;
+time: INTEGER;
+delta: INTEGER;
```

Comment:
This was now renamed to `wheel_delta`, sorry :p

Reply:
Renamed.